library/std/src/Std/Intrinsic.qs

```diff
@@ -1140,6 +1140,58 @@ operation Z(qubit : Qubit) : Unit is Adj + Ctl {
     adjoint self;
 }
 
+/// # Summary
+/// Applies the given unitary matrix to the given qubits. The matrix must be square, with dimensions dictated by the number of qubits.
```

Comment:
Unitary matrix is always square. I suggest leaving only the first sentence in the summary and consolidating second sentence and remarks in description. Then removing remarks. As stated in readme, remarks are for "General knowledge that may be useful for understanding of the functionality provided."

Reply:
I guess I was assuming not everyone knows a priori that unitary requires square so was trying to explain that, but given this is already a fairly advanced utility it's probably safe to let the terminology do the heavy lifting. I'll make the changes you suggest.